web/models.py

```
@@ -1752,6 +1753,65 @@ def display_name(self):
+    def can_edit(self, user: User) -> bool:
+        """Check if the given user can edit this meetup."""
+        return user == self.creator or user.is_staff
+
+
+class MeetupRegistration(models.Model):
+    meetup = models.ForeignKey(Meetup, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    registered_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ["user", "meetup"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)